src/schema/builders/WalletsCsv.ts

```diff
@@ -0,0 +1,35 @@
+import path from "path"
+
+// Expand the WalletsCsv type to have a new `image` field with its corresponding
+// asset image
+export const WalletsCsv = {
+  name: "WalletsCsv",
+  interfaces: ["Node"],
+  extensions: {
+    infer: true,
+  },
+  fields: {
+    image: {
+      type: "File",
+      resolve: async (source, _args, context) => {
```

Comment:
Rad 🤩

Reply:
I think we'll just need to make clear that we need to save wallet images as `${source.id}.png`.

Reply:
🤯

Reply:
Yea this is pretty cool from Gatsby where you can customize the schema. More on this here: https://www.gatsbyjs.com/docs/reference/graphql-data-layer/schema-customization/#foreign-key-fields

> I think we'll just need to make clear that we need to save wallet images as `${source.id}.png`.

Agree. Where do you think we should add more doc? in this file or in a new doc file under `/docs`?

Reply:
Perhaps a new file under `/docs` makes sense around schema? We could add this note as well as the layout structure you described here: https://github.com/ethereum/ethereum-org-website/pull/6486#issuecomment-1146411041

Reply:
Otherwise a md file in the directory https://github.com/ethereum/ethereum-org-website/tree/ts-pages-conditional/src/schema may make sense so if people navigate to it e.g. via GH.

Reply:
I suppose I'd favor in `/docs` to keep those doc resources in one place.

Reply:
Yes, to keep things consistent and easy to find I will go with the `/docs`.